src/app/views/query-response/snippets/Snippets.tsx

```diff
@@ -28,7 +28,11 @@ function GetSnippets() {
     'JavaScript': {
       sdkDownloadLink: 'https://aka.ms/graphjssdk',
       sdkDocLink: 'https://aka.ms/sdk-doc'
-    }
+    },
```

Comment:
I believe you are talking about the ordering here @adhiambovivian.
We automatically order the snippet tabs before they are rendered. So it doesn't matter where they are placed in this object